src/common/murmurhash2.h

```diff
@@ -0,0 +1,22 @@
+//-----------------------------------------------------------------------------
+// MurmurHash2 was written by Austin Appleby, and is placed in the public
+// domain. The author hereby disclaims copyright to this source code.
```

Comment:
This is nice, but should we vendor this in thirdparty library or vendor at somewhere? @PragmaTwice

Reply:
I think currently it is ok to put it in `common/`, but we may move it to a vendor directory later.